inspirehep/modules/theme/views.py

```diff
@@ -350,7 +350,10 @@ def ping():
 
 @blueprint.route('/ajax/references', methods=['GET'])
 def ajax_references():
-    """Handler for datatables references view"""
+    """Handler for datatables references view.
+
+    DEPRECATED: REMOVE WHEN THE UI IS RELEASED.
```

Comment:
Nit: the sphinx way to do it is like that https://github.com/inspirehep/inspire-next/blob/75d6bcdf28e1c0cb5e4e773a7af4f2e6157b1831/tests/ui/conftest.py#L88